.github/workflows/neuron-ci.yml

```diff
@@ -152,7 +152,7 @@ jobs:
               if [ "$NRN_ENABLE_PYTHON_DYNAMIC" == "ON" ]; then
                 export CMAKE_OPTION="-DNRN_ENABLE_PYTHON=ON -DNRN_ENABLE_PYTHON_DYNAMIC=ON -DNRN_PYTHON_DYNAMIC=${PYTHON_MIN};${PYTHON_MAX} -DNRN_ENABLE_CORENEURON=ON";
               else
-                export CMAKE_OPTION="$CMAKE_OPTION -DPYTHON_EXECUTABLE=${PYTHON}";
+                export CMAKE_OPTION="$CMAKE_OPTION -DPython_EXECUTABLE=${PYTHON}";
```

Comment:
@alexsavulescu : just mentioning - if this is not straightforward and going to introduce newer CMake variable (capital vs small, which will be more confusing / fragmented ), may be its not worth to make this change in the upcoming 8.1 release?

Reply:
definitely. Let's consider this PR to be just exploration :)

Reply:
Sure!